assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -110,10 +109,15 @@ export default function WPDashboardPopularPages( props ) {
 		return <WidgetReportError moduleSlug="analytics" error={ error } />;
 	}
 
-	if ( isZeroReport( report ) && ! zeroDataStatesEnabled ) {
+	if ( isGatheringData && ! zeroDataStatesEnabled ) {
```

Comment:
This prevents showing the gathering data message when the site is not gathering data.

**Before**
![image](https://user-images.githubusercontent.com/1621608/160181059-4856bf24-30da-47e1-b4c4-1654da9d4e7e.png)

**After**
![image](https://user-images.githubusercontent.com/1621608/160181077-65253bf2-307d-4b59-b5f9-d4eda7491fbf.png)